repo-projects/docker-creds-provider.proj

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <DockerCredsProviderReleaseVersion>2.2.1</DockerCredsProviderReleaseVersion>
+    <DockerCredsProviderReleaseVersion>2.2.4</DockerCredsProviderReleaseVersion>
```

Comment:
This will require a corresponding update in the sdk repo that references this.

Reply:
Yes, I was planning to make that change in the SBE flow PR.